en/dev_setup/dev_env_linux_ubuntu.md

```diff
@@ -2,22 +2,25 @@
 
 The following instructions set up a PX4 development environment on Ubuntu Linux 18.04 and 20.04.
 This environment can be used to build PX4 for [most PX4 targets](../dev_setup/dev_env.md#supported-targets):
-* Pixhawk and other NuttX-based hardware
-* [Gazebo Simulation](../sim_gazebo_gz/README.md) (Ubuntu 20.04 and later)
-* [Gazebo Classic Simulation](../sim_gazebo_classic/README.md) (up to Ubuntu 20.04)
-* [jMAVSim Simulation](../simulation/jmavsim.md)
-* [Raspberry Pi](#raspberry-pi)
-* [ROS (1)](#ros-gazebo) (Robotics Operating System)
+
+- Pixhawk and other NuttX-based hardware
+- [Gazebo Simulation](../sim_gazebo_gz/README.md) (Ubuntu 20.04 and later)
```

Comment:
What I will do is change the line "3" about to note that generally the toolcahin isn't fully supported in 22.04. I'm not going to mention 18.04 here, because I don't want people using 18.04 and it isn't particularly a target for Gazebo Garden.